packages/electrode-archetype-react-app/lib/logger.js

```diff
@@ -0,0 +1,23 @@
+const winston = require("winston");
```

Comment:
what about when winston is not installed because app did `npm install --production` which would skip installing the dev archetype.